apps/jobs/models.py

```diff
@@ -129,6 +129,7 @@ class Submission(TimeStampedModel):
     ignore_submission = models.BooleanField(default=False)
     # Store the values of meta attributes for the submission here.
     submission_metadata = JSONField(blank=True, null=True)
+    is_verified = models.BooleanField(default=False)
```

Comment:
`is_verified` -> `is_verified_by_host`. Also, please add comments on why this field is there.